repository/v.json

```diff
@@ -779,20 +779,25 @@
 			]
 		},
 		{
-			"name": "Vue Syntax Highlight",
-			"details": "https://github.com/vuejs/vue-syntax-highlight",
+			"name": "Vue",
+			"details": "https://github.com/SublimeText/Vue",
+			"previous_names": ["Vue Syntax Highlight"],
 			"releases": [
 				{
 					"sublime_text": "<3153",
 					"tags": "oldsyntax-"
 				},
 				{
-					"sublime_text": "3153 - 3999",
+					"sublime_text": "3153 - 4106",
 					"tags": "newsyntax-"
 				},
 				{
-					"sublime_text": ">4000",
-					"tags": "st4-"
+					"sublime_text": "4107 - 4125",
+					"tags": "4107-"
+				},
+				{
+					"sublime_text": ">=4126",
+					"tags": "4126-"
```

Comment:
Just a note that such a tag does not exist yet. We will create `4126-5.0.0` once we are done with the planned updates, including an update message regarding the package rename and takeover as well as the changes.

Reply:
Yes, tags/releases haven't been made yet on purpose, until we are sure we got everything we want.

Reply:
5.0.0 has been released: https://github.com/SublimeText/Vue/releases/tag/4126-5.0.0